client/lib/analytics/ad-tracking.js

```diff
@@ -780,6 +802,22 @@ function recordProduct( product, orderId ) {
 			}
 		}
 
+		// Facebook
+		if ( isFacebookEnabled ) {
```

Comment:
An idea on this one: in case Abraham wants the products individually, but Ori doesn't, then this can be changed in `if (isFacebookEnabled && isJetpackProduct)` and I guess things will be fine quickly.

Reply:
Thank you. I checked with Ori and I'm excluding the per-product reporting for now. We can revisit in the future if per-product reporting becomes desirable. For now I am commenting that section of code out and noting the issue that you raised here.